### PR TITLE
feat(hosting): remove ownlogs deprecated routes

### PR DESCRIPTION
**Pull request by lucas-chaigne**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | WEB-17359
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

goal is to get rid of deprecated routes calls in manager : 
/hosting/web/{serviceName}/userLogs GET and POST
/hosting/web/{serviceName}/userLogs/{login} : GET, PUT, POST
/hosting/web/{serviceName}/userLogs/{login}/changePassword : POST

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js [hunk 1]
```diff
@@ -528,23 +528,41 @@ import union from 'lodash/union';
...
-      getUserLogs(serviceName) {
-        return this.OvhHttp.get(`/hosting/web/${serviceName}/userLogs`, {
+      getOwnLogs(serviceName, fqdn) {
+        return this.OvhHttp.get(`/hosting/web/${serviceName}/ownLogs`, {
           rootPath: 'apiv6',
+          params: {
+            fqdn,
+          },
         });
       }
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js:542]:
```suggestion
      getOwnLogs(serviceName,id) {
        return this.OvhHttp.get(`/hosting/web/${serviceName}/ownLogs`, {
          rootPath: 'apiv6',
          params: {
           id,
          },
        });
      }
```

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js [hunk 2]
```diff
@@ -600,30 +620,35 @@ import union from 'lodash/union';
...
        * @param {string} password
        */
-      userLogsCreate(serviceName, description, login, password) {
-        return this.OvhHttp.post(`/hosting/web/${serviceName}/userLogs`, {
-          rootPath: 'apiv6',
-          data: {
-            description,
-            login,
-            password,
+      userLogsCreate(serviceName, description, id, login, password) {
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js:627]:
```suggestion
      userLogsCreate(serviceName, id, description, login, password) {
```

**Review by sachinrameshn — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js [hunk 3]
```diff
@@ -89,19 +89,30 @@ angular.module('App').controller(
         type: 'action',
       });
       this.$scope.resetAction();
-      return this.Hosting.userLogsCreate(
+
+      this.Hosting.getOwnLogs(
+        this.$stateParams.productId,
         this.$stateParams.productId,
```
Comment (sachinrameshn) [inline packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js:95]: Is productId should be sent twice ?

File: packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js (hunk 1 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js:542]: param is fqdn for this route : https://api.ovh.com/console/#/hosting/web/%7BserviceName%7D/ownLogs~GET

File: packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js (hunk 2 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/dashboard/hosting.service.js]: I reordered the params of the function to match apiv6

File: packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js (hunk 3 above)
Comment (JacquesLarique) [inline packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js:95]: Can't we use just one parameter ?

Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/user-logs/add/hosting-user-logs-add.controller.js:95]: the api to get ownLog takes 2 parameters : the serviceName (to get ownLogs for the hosting) and a filter to get ownLog on a particular domain name of the hosting. Here we want the default own log of the hosting, so we filter on the main domain name which is the service name.

**Review by darsene — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

**Review by antonymarion — APPROVED** [review]

**Pull request merged**